playground/frontend/playground_components/lib/playground_components.dart

```diff
@@ -55,7 +56,6 @@ export 'src/theme/switch_notifier.dart';
 export 'src/theme/theme.dart';
 export 'src/util/pipeline_options.dart';
 export 'src/util/string.dart';
-
```

Comment:
This was made on purpose. Maybe add more blank lines between different root directories?

Reply:
blank lines automatically deletes on imports optimizing

Reply:
Then please do not optimize imports automatically in this file. Every useful change that could do is highlighted by linter anyway.